From 6bd516a8f1a1ee91fd2c5593083b73f6a50c2127 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 09 Jan 2020 14:43:20 +0200
Subject: [PATCH] [SW-7481] gaudi: pll: remove hbm configuration

The HBM PLL configuration is now done by U-Boot and therefore we remove
it from preboot - no need to do this twice.

Change-Id: I3a5452961aec75a3b8908ded5907181f82205e37
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index 7c0fe3b..a15bd63 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -118,15 +118,6 @@
 			     HL_PLL_DIV_PLL
 		},
 	}, {
-		.name = "HBM", .base = mmPSOC_HBM_PLL_BASE,
-		.nr = 0, .nf = 63, .od = 3, .nb = 31, /* 800 MHz */
-		.div_factor = { 0, 1, -1, -1 },
-		.div_sel = { HL_PLL_PLL_CLK,
-			     HL_PLL_DIV_PLL,
-			     HL_PLL_REF_CLK,
-			     HL_PLL_REF_CLK
-		},
-	}, {
 		.name = "NIC0", .base = mmNIC0_PLL_BASE,
 		.nr = 24, .nf = 1665, .od = 3, .nb = 832, /* 833 MHz */
 		.div_factor = { 0, 4, 8, -1 },
@@ -142,15 +133,6 @@
 		.div_sel = { HL_PLL_PLL_CLK,
 			     HL_PLL_DIV_PLL,
 			     HL_PLL_DIV_PLL,
-			     HL_PLL_REF_CLK
-		},
-	}, {
-		.name = "NIC2_HBM", .base = mmNIC2_HBM_PLL_BASE,
-		.nr = 0, .nf = 63, .od = 3, .nb = 31, /* 800 MHz */
-		.div_factor = { 0, 1, -1, -1 },
-		.div_sel = { HL_PLL_PLL_CLK,
-			     HL_PLL_DIV_PLL,
-			     HL_PLL_REF_CLK,
 			     HL_PLL_REF_CLK
 		},
 	}, {
diff --git a/gaudi/common.h b/gaudi/common.h
index 4ef9711..a9fe4d5 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -22,9 +22,7 @@
 #define mmPSOC_MME_PLL_BASE		0xC71000
 #define mmPSOC_PCI_PLL_BASE		0xC72000
 #define mmPSOC_TPC_PLL_BASE		0xC73000
-#define mmPSOC_HBM_PLL_BASE		0xC74000
 #define mmNIC0_PLL_BASE			0xCF9000
 #define mmNIC1_PLL_BASE			0xD39000
-#define mmNIC2_HBM_PLL_BASE		0xD78000
 #define mmNIC2_MME_PLL_BASE		0xD79000
 #define mmNIC2_TPC_PLL_BASE		0xD7A000
